hack/ssh.sh

```diff
@@ -11,5 +11,9 @@ ID=$3
 
 IP=$(az vmss list-instance-public-ips -g $RESOURCEGROUP -n $SS --query "[$ID].ipAddress" | tr -d '"')
 
+scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
```

Comment:
Note: unrelated to MSI - this is to allow you to ssh from master to other nodes. It tripped me up :smile: 

@kwoodson might just want to put this in its own commit or PR

Reply:
@pweil-, ok, i can do that.